cloudigrade/util/aws/sqs.py

```diff
@@ -39,6 +39,9 @@ def delete_message_from_queue(queue_url, messages):
         dict: The response from the delete call.
 
     """
+    if not bool(messages):
```

Comment:
Why are you explicitly casting this to a `bool`? `if not messages` should work for an empty list.

Reply:
I got curious about this too. But asked on Slack. Better to talk here instead.

Reply:
I can remove that.  Great catch.